pkg/agent/runners/checktcpport.go

```diff
@@ -12,8 +12,8 @@ import (
 	"time"
 
 	"github.com/gardener/network-problem-detector/pkg/common/config"
-
```

Comment:
Please re-add the line so that `golangci-lint` does not complain (fixing the `verify` job).